README.md

```diff
@@ -13,6 +13,10 @@ Feel free to contact our partnership team (partnerships@valitor.com) if you need
 
 ## Change log
 
+3.1.2
+* Bug fixes
+    - Fixed issue for subscription payments
```

Comment:
```suggestion
    - Use `subscription` as auth type instead of `recurring`
```

Reply:
Either be specific about the change, or be specific about the fix. "Issue for subscription" can be tricky when you have a change log with 10+ versions and release notes.